src/fixed.rs

```diff
@@ -0,0 +1,90 @@
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use ffi;
+use glib::translate::*;
+
+use Container;
+use ContainerExt;
+use Fixed;
+use IsA;
+use Value;
+use Widget;
+
+fn has_widget<O: IsA<Fixed> + IsA<Container>, T: IsA<Widget>>(c: &O, item: &T) -> bool {
+    skip_assert_initialized!();
+    let t: *mut ffi::GtkContainer = c.to_glib_none().0;
+    let t: Container = unsafe { from_glib_none(t) };
+    let widgets = t.get_children();
+    let this: *mut ffi::GtkWidget = item.to_glib_none().0;
+    widgets.iter().find(|w| {
+        let w: *mut ffi::GtkWidget = w.to_glib_none().0;
+        w == this
+    }).is_some()
+}
+
+pub trait FixedExtManual {
+    fn get_child_x<T: IsA<Widget>>(&self, item: &T) -> i32;
+
+    fn set_child_x<T: IsA<Widget>>(&self, item: &T, x: i32);
+
+    fn get_child_y<T: IsA<Widget>>(&self, item: &T) -> i32;
+
+    fn set_child_y<T: IsA<Widget>>(&self, item: &T, y: i32);
+}
+
+impl<O: IsA<Fixed> + IsA<Container>> FixedExtManual for O {
+    fn get_child_x<T: IsA<Widget>>(&self, item: &T) -> i32 {
+        assert!(has_widget(self, item), "this item isn't in the Fixed's widget list");
+        let mut value = Value::from(&0);
+        unsafe {
+            ffi::gtk_container_child_get_property(self.to_glib_none().0, item.to_glib_none().0, "x".to_glib_none().0, value.to_glib_none_mut().0);
+        }
+        value.get().unwrap()
+    }
+
+    fn set_child_x<T: IsA<Widget>>(&self, item: &T, x: i32) {
+        assert!(has_widget(self, item), "this item isn't in the Fixed's widget list");
+        unsafe {
+            ffi::gtk_container_child_set_property(self.to_glib_none().0, item.to_glib_none().0, "x".to_glib_none().0, Value::from(&x).to_glib_none().0);
+        }
+    }
+
+    fn get_child_y<T: IsA<Widget>>(&self, item: &T) -> i32 {
+        assert!(has_widget(self, item), "this item isn't in the Fixed's widget list");
+        let mut value = Value::from(&0);
+        unsafe {
+            ffi::gtk_container_child_get_property(self.to_glib_none().0, item.to_glib_none().0, "y".to_glib_none().0, value.to_glib_none_mut().0);
+        }
+        value.get().unwrap()
+    }
+
+    fn set_child_y<T: IsA<Widget>>(&self, item: &T, y: i32) {
+        assert!(has_widget(self, item), "this item isn't in the Fixed's widget list");
+        unsafe {
+            ffi::gtk_container_child_set_property(self.to_glib_none().0, item.to_glib_none().0, "y".to_glib_none().0, Value::from(&y).to_glib_none().0);
+        }
+    }
+}
+
+#[test]
+#[should_panic]
+fn test_fixed_fail() {
+    use init;
+    use Button;
+
+    if init().is_err() {
+        return;
+    }
+
+    let container = Fixed::new();
+
+    let button1 = Button::new_with_label("Button 1");
+    let button2 = Button::new_with_label("Button 2");
+    container.add(&button1);
+
+    let _ = container.get_child_y(&button1);
+    // Should panic here.
+    let _ = container.get_child_y(&button2);
+}
```

Comment:
This test will panic on `init` on travis, so better removed it too.
I will add test to https://github.com/EPashkin/gtk-rs-runtime-tests.

Reply:
Oh nice!